models/topics_models.js

```diff
@@ -7,6 +7,22 @@ function fetchTopics() {
     })
 }
 
+function fetchArticles() {
+    const sqlStr = `SELECT articles.author, articles.title, articles.article_id,
+    articles.topic, articles.created_at, articles.votes, articles.article_img_url, 
+    COUNT(comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments
```

Comment:
I would stick with async await or promises - we're doing both between the controller and model

Reply:
Hey Pippa! Thanks for your feedback! I did attempt this but for some reason async/await in the models was failing a couple of test, so I decided to keep async await on controllers whilst keeping promises in the models.